dts/bindings/spi/nxp,kinetis-dspi.yaml

```diff
@@ -41,3 +41,61 @@ properties:
     pinctrl-0:
       type: phandles
       required: true
+
+    nxp,rx-tx-chn-share:
+      type: boolean
+      required: false
+      description: If the edma channel shared with tx and rx
+
+    pcs:
+      type: int
+      required: false
+      description: |
+        pcs slection range form 0-4
```

Comment:
This the chip select, why do we need a new property?

Reply:
right,I mis-undsertand the config->slave, should use this.